### PR TITLE
Add new types for Meteor Atmosphere package littledata:synced-cron

### PR DESCRIPTION
**Pull request by ghybs**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package. **=> actually compatible with the same name npm package**
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/meteor-synced-cron/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
+// Type definitions for littledata:synced-cron 1.5
```
Comment (peterblazejewicz) [inline types/meteor-synced-cron/index.d.ts:1]:
```suggestion
// Type definitions for non-npm package littledata:synced-cron 1.5
```

File: types/meteor-synced-cron/tslint.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```
Comment (peterblazejewicz) [inline types/meteor-synced-cron/tslint.json:4]:
```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```

File: types/meteor-synced-cron/index.d.ts (hunk 1 above)
Comment (ghybs) [reply types/meteor-synced-cron/index.d.ts:1]: Hi @peterblazejewicz,

Thank you so much for the review and change suggestion!

Looks like I did not understand the dts lint error properly... 😕 

Changing to this raises another error, so I will accept the change and fix it after.

Thanks again!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@ghybs thanks!

**Comment by ghybs** [comment]
Ready to merge

**Pull request merged**